src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,36 @@
+package duke.task;
+
+/**
+ * A task that occurs at a location that has to be done.
+ */
+public class Event extends Task {
+
+    private String at;
```

Comment:
Shouldn't the variable name be more descriptive? Instead of at maybe you can try eventDateTime?

Reply:
I changed it to `eventLocation`.